src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectMessage.cs

```diff
@@ -39,17 +39,25 @@ namespace Microsoft.IdentityModel.Protocols.OpenIdConnect
     /// </summary>
     public class OpenIdConnectMessage : AuthenticationProtocolMessage
     {
-        private const string _skuTelemetryValue = "ID_NET";
 
         /// <summary>
         /// Initializes a new instance of the <see cref="OpenIdConnectMessage"/> class.
         /// </summary>
-        public OpenIdConnectMessage() { }
+        public OpenIdConnectMessage()
+        {
+#if NET45
+            SkuTelemetryValue = "ID_NET45";
+#elif NET451
+            SkuTelemetryValue = "ID_NET451";
+#elif NETSTANDARD1_4
+            SkuTelemetryValue = "ID_NETSTANDARD1_4";
+#endif
+        }
 
         /// <summary>
         /// Initializes an instance of <see cref="OpenIdConnectMessage"/> class with a json string.
         /// </summary>
-        public OpenIdConnectMessage(string json)
+        public OpenIdConnectMessage(string json): this()
```

Comment:
normally we put :this() on a new line, indented.

Reply:
Done! Though I'm not entirely sure what the syntax should look like since I can't find an example of assigning a default value to a property along with #if statements.